packages/fluentui/react-bindings/test/hooks/useTriggerElement-test.tsx

```diff
@@ -24,7 +24,7 @@ describe('useTriggerElement', () => {
   });
 
   it('"trigger" can be null', () => {
-    const wrapper = mount(<TestComponent trigger={null} />);
+    const wrapper = mount(<TestComponent trigger={null as any} />);
```

Comment:
I would like to check this, as it's not expected

Reply:
`UseTriggerElementOptions` currently specifies `trigger` as `trigger?: React.ReactElement`. I guess `null` was previously accepted for optional props, but the new TS version got stricter and doesn't allow that?

Reply:
`packages/fluentui/react-bindings/src/hooks/useTriggerElement.tsx` line 9

Just modify that line to `trigger?: React.ReactElement | null;`

Reply:
The problem is that the test props type doesn't match the actual prop type